README.md

```diff
@@ -1,20 +1,33 @@
 video-mode-plugin
 =================
 
-A BrightAuthor plugin that changes the video resolution, frame rate, color space and color depth.
+A BrightSign plugin that changes the video resolution, frame rate, color space and color depth.
 
-# Usage
+SetSingleScreenVideoMode.brs is used for series 4 and below.
+SetMultiScreensVideoMode.brs is used for series 5 players.
+videomode_plugin.brs is a alternative example that you can configure videoMode for series 4 and below via Plugin Message Usage. 
 
-1. Add videomode_plugin.brs to BrightAuthor (File > Presentation Properties > Autorun)
-1. Add a plugin message wherever you want to send it: Advanced tab > Add Command > Send > Send Plugin Message, pick the plugin you added in step 1, and add the text of the the plugin message e.g. "brightsign!videomode!1920x1080x60i!444!10bit" (format below)
+# VideoMode Plugin Usage
+
+Add SetSingleScreenVideoMode.brs or SetMultiScreensVideoMode.brs to BrightAuthor or BrightAuthor:Connected (Presentation Settings > Support Content > Video Mode Plugin)
```

Comment:
Feels to me the first step should be update SetSingleScreenVideoMode.brs or SetMultiScreensVideoMode.brs with the user's screens settings.